src/Header.svelte

```diff
@@ -162,15 +163,30 @@
 
   <div class="right">
     {#if wallet && wallet.network.name === "goerli"}
-      <a use:link href="/faucet">
+      <!-- svelte-ignore a11y-invalid-attribute -->
```

Comment:
Can we just disable this lint? It's creating a lot of noise.

Reply:
I'm not aware of an easy way to [disable these globally](https://github.com/radicle-dev/radicle-interface/issues/449). A potential workaround is to use a different html tag, or extract it into an `<InternalLink>` component.

Reply:
Ok we have removed all the `invalid-attribute` tags and there is a PR on the way to remove some more wrt to the missing click events #471